test/test_support.ts

```diff
@@ -38,8 +53,17 @@ export const compilerOptions: ts.CompilerOptions = {
   emitDecoratorMetadata: true,
   noEmitHelpers: true,
   jsx: ts.JsxEmit.React,
-  // Flags below are needed to make sure source paths are correctly set on write calls.
-  rootDir: path.resolve(process.cwd()),
+  /**
+   * The 'conceptual path' of source files in TypeScript is their path relative to the current
```

Comment:
This term is ill-defined (which I'm sure you're aware of). Is it a bazel concept, a TypeScript concept, or something else?  Maybe a Closure-only concept?